expose.go

```diff
@@ -0,0 +1,111 @@
+package chromedp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/chromedp/cdproto/page"
+	"github.com/chromedp/cdproto/runtime"
+)
+
+// ExposedFunc is the function type that can be exposed to the browser env.
+type ExposedFunc func(args string) (string, error)
+
+// ExposeAction are actions which expose Go functions to the browser env.
+type ExposeAction Action
+
+// Expose is an action to add a function called fnName on the browser page's
+// window object. When called, the function executes fn in the Go env and
+// returns a Promise which resolves to the return value of fn.
+//
+// Note:
+// 1. This is the lite version of puppeteer's [page.exposeFunction].
+// 2. It adds "chromedpExposeFunc" to the page's window object too.
+// 3. The exposed function survives page navigation until the tab is closed?
```

Comment:
yes, no way to revert this action.  
```
`Runtime.removeBinding` does not remove binding function from global object 
but unsubscribes current runtime agent from Runtime.bindingCalled notifications.
```

Reply:
Not only the binding, but also the scripts that will be evaluated on new document (`page.AddScriptToEvaluateOnNewDocument`). I know that we can store the `ScriptIdentifier` and remove it later, but it will make the code complicated.

Reply:
It only takes effect on the page after navigation. The target(Page) hasn't changed. this kind of behavior should be understood

Reply:
Most state does not survive page navigation. But the scripts that the expose action uses survive page navigation and there is no way to stop that (unless the tab is closed), so this behavior should be documented.

Reply:
I get it.